frontend/src/common/theme.ts

```diff
@@ -29,7 +29,16 @@ export const inter = Inter({
   variable: "--font-inter",
 });
 
-const { fontWeights } = defaultAppTheme;
+/**
+ * (masoudmanson): IMB_Plex_Mono is needed for SDS code blocks.
+ */
+export const ibm_plex_mono = IBM_Plex_Mono({
+  subsets: ["latin"],
+  variable: "--font-ibm-plex-mono",
+  weight: ["400", "600"],
+});
+
+const { fontWeights } = SDSAppTheme;
 
 const iconSizes = {
```

Comment:
The icon sizes have been updated on the SDS side, so you'll need to make updates in the theme file as well.

```javascript
const iconSizes = {
  input: { height: 16, width: 16 }, // for use with input icons only (radio and checkbox)
  l: { height: 24, width: 24 },
  m: { height: 16, width: 16 }, // (masoudmanson): SDS doesn't have a medium icon size!
  s: { height: 16, width: 16 }, // (masoudmanson): Used to be 12px
  xl: { height: 32, width: 32 },
  xs: { height: 12, width: 12 }, // (masoudmanson): Used to be 8px
};
```